### PR TITLE
feat(frontend/proposal_list): Add reviewed and unreviewed filter

### LINKED ISSUE #239: Add reviewed and unreviewed filter along with section for proposal page
In dashboard page we display section with reviewed and unreviewed number . We can't add hyperlink to pass filter or flag to filter reviewed and unreviewed proposal in url like proposal_type=2 or proposal_section=5.
Proposal is to add allow add filter like proposal_section=5&reviewed=1 and proposal_section=5&reviewed=0.

### PR DESCRIPTION
**Pull request by anistark**
a porro conference proposals

closes #239

### CODE REVIEW

**Comment by theskumar** [comment]
@anistark please open a new PR for coverall separately, you can remove the commit w.r.t coverall in this PR.

**Comment by anistark** [comment]
@theskumar Removed coveralls. Adding separate PR for coveralls.

File: junction/templates/proposals/partials/proposal-list--items.html [hunk 1]
```diff
@@ -46,6 +46,14 @@ <h3 class="proposal--title">
                         <a class='tag tag__proposal_section label label-proposal-type inline-block'
                             href="?proposal_section={{proposal.proposal_section.id}}">{{ proposal.proposal_section }}
                         </a>
+                        <a class='tag tag__proposal_section label label-proposal-type inline-block'
+                            href="?proposal_status={{proposal.review_status}}">
+                            {% if proposal.review_status == 1 %}
+                                Reviewed
+                            {% elif proposal.review_status == 1 %}
```
Comment (theskumar) [inline junction/templates/proposals/partials/proposal-list--items.html]: Isn't the `if` and `elif` same here? I think you'll also need to write some python code in the view function to handle this, thoughts?

Comment (theskumar) [inline junction/templates/proposals/partials/proposal-list--items.html]: https://github.com/pythonindia/junction/blob/master/junction/proposals/views.py#L75-L77

Comment (anistark) [inline junction/templates/proposals/partials/proposal-list--items.html]: @theskumar Yes. Needs to handle the filtering too. Missed that. I thought earlier that it only needs to handle in the frontend to display.

**Comment by anistark** [comment]
@theskumar Please review now.

File: requirements-dev.txt [hunk 2]
```diff
@@ -8,7 +8,7 @@ flake8==2.4.0
 pytest-django==2.8.0
 pytest-flakes==0.2
 pytest-pythonpath==0.6
-git+git://github.com/mverteuil/pytest-ipdb.git
+# git+git://github.com/mverteuil/pytest-ipdb.git
```
Comment (theskumar) [inline requirements-dev.txt]: Please do not change things that do not relate to the PR you have open. You can always create a new issue/PR for changing such things. You can undo this change.

File: junction/proposals/models.py [hunk 3]
```diff
@@ -59,6 +59,19 @@ def __str__(self):
...
+class ProposalStatus(AuditModel):
+
+    """ List of Proposal Types """
+    name = models.CharField(max_length=255, verbose_name="Proposal Status Name")
+    description = models.TextField(default="")
+    active = models.BooleanField(default=True, verbose_name="Is Active?")
+    conferences = models.ManyToManyField(to=Conference, related_name='proposal_status')
+
+    def __str__(self):
+        return self.name
```
Comment (theskumar) [inline junction/proposals/models.py]: Why do we need this model? If we wan to know, if the proposal is reviewed or not, we can just add a new property `is_reviewed` to model to `Proposal` model class itself. Thoughts?

**Comment by theskumar** [comment]
seems like there are change in the master branch, you should rebase your work on top of it before continuing. The way to do it is:
```
git remote add upstream git@github.com:pythonindia/junction.git
git fetch upstream
git checkout master
git rebase upstream/master
git push origin master --force
```

Assuming running `git remote -v` indicates location of `origin` to your forked repository. On a side note, while working on something, you should always create a new branch with a descriptive name[1] and then send a pull request from that branch. 

[1] https://github.com/agis-/git-style-guide

File: requirements-dev.txt (hunk 2 above)
Comment (anistark) [inline requirements-dev.txt]: Oh sorry about this. I'll undo this ASAP.

**Comment by kracekumar** [comment]
Since the feature is already in master, closing the PR.
